features/FEATURE_BLE/ble/gap/ScanParameters.h

```diff
@@ -175,7 +176,17 @@ class ScanParameters {
     {
         return scanning_filter_policy;
     }
+#else
+    ScanParameters &setFilter(scanning_filter_policy_t filter_policy))
+    {
+#error "Whitelist feature disabled"
```

Comment:
that is not correct as it will trigger a compile time error even if the function is not called.

Reply:
yep, fixed